CRM/Upgrade/Incremental/php/FiveFiftyThree.php

```diff
@@ -21,6 +21,16 @@
  */
 class CRM_Upgrade_Incremental_php_FiveFiftyThree extends CRM_Upgrade_Incremental_Base {
 
+  public function setPostUpgradeMessage(&$postUpgradeMessage, $rev) {
+    if ($rev === '5.53.alpha1') {
+      $postUpgradeMessage .= '<br/>' . ts("WARNING: CiviCRM has changed our the date format variable %1 is outputted when using CRM_Core_Date::customFormat / crmDate.  Please review your <a href='%2' target='_blank'>Date Format</a> settings and your <a href='%3' target='_blank'>system message templates</a> for usage of the %1 variable", [
```

Comment:
> CiviCRM has changed our the date format variable

our => how ?

Also CRM_Utils_Date not Core_Date. But I think I'll put up an alternate PR because while a message is still good, it seems safe to just go ahead and have the upgrade replace %A with %P in any of the settings on the admin page, and for message templates it could tell you which ones have the string %A in them.

Reply:
The tricky part is that if it is `date_format|%A` is fine but not `crmDate|%A`

Reply:
For the settings on the admin page they don't have smarty code in them and are purely format strings, so I think it's safe to just replace there. In message templates yes there might be all kinds of whack.